rotkehlchen/db/dbhandler.py

```diff
@@ -1546,18 +1547,26 @@ def get_manually_tracked_balances(
         data = []
         for entry in query:
             tags = deserialize_tags_from_db(entry[4])
+            if (asset_is_missing := not Asset(entry[0]).exists()) is True:
+                self.msg_aggregator.add_warning(
```

Comment:
Shouldn't the frontend handle this? Why add a warning that will be shown to the frontend here?